### PR TITLE
Dilpreert | Test | Reported Issue with sync for stage and test

### PR DESCRIPTION
**Pull request by dilpreetsinghofficial**
* **What kind of change does this PR introduce?** (Bug fix, feature, docs update, ...)

* **What is the current behavior?** (You can also link to an open issue here)

* **What is the new behavior (if this is a feature change)?**

* **Other information**:

### CODE REVIEW

**Review by dvCodeWorld — APPROVED** [review]

**Review by dvCodeWorld — APPROVED** [review]

File: apps/web-giddh/src/app/subscription/buy-plan/buy-plan.component.html [hunk 1]
```diff
@@ -549,35 +552,35 @@ <h4 class="font-16 strong d-flex align-items-center mb-3 plan-summary-heading">
...
-                                                    [name]="'billingName'"
+                                                    [name]="'subscriptionBillingName' + currentTimestamp"
                                                     [label]="localeData?.billing_name + '*'"
                                                     [placeholder]="localeData?.billing_name"
                                                     formControlName="billingName"
                                                     [showError]="
                                                         isFormSubmitted && !secondStepForm.get('billingName')?.value
                                                     "
-                                                    [autocomplete]="'off'"
+                                                    [autocomplete]="'subscriptionBillingName' + currentTimestamp"
```
Comment (ravinderwalkover) [inline apps/web-giddh/src/app/subscription/buy-plan/buy-plan.component.html]: what's the use of this????

File: apps/web-giddh/src/app/subscription/buy-plan/buy-plan.component.ts [hunk 2]
```diff
@@ -184,6 +184,8 @@ export class BuyPlanComponent implements OnInit, OnDestroy {
     public subscriptionRazorpayOrderDetails$ = this.componentStore.select(state => state.subscriptionRazorpayOrderDetails);
     /** True if it is subscription region */
     public isSubscriptionRegion: boolean = false;
+    /** Hold current time stamp  */
+    public currentTimestamp: string;
```
Comment (ravinderwalkover) [inline apps/web-giddh/src/app/subscription/buy-plan/buy-plan.component.ts]:
```suggestion
    public currentTimeStamp: string;
```

File: apps/web-giddh/src/app/subscription/change-billing/change-billing.component.ts [hunk 3]
```diff
@@ -88,12 +89,15 @@ export class ChangeBillingComponent implements OnInit, OnDestroy {
         billingName: "",
         uniqueName: ""
     };
+    /** Hold current time stamp  */
+    public currentTimestamp: string;
```
Comment (ravinderwalkover) [inline apps/web-giddh/src/app/subscription/change-billing/change-billing.component.ts]:
```suggestion
    public currentTimeStamp: string;
```

File: apps/web-giddh/src/app/subscription/change-billing/change-billing.component.ts [hunk 4]
```diff
@@ -110,6 +114,7 @@ export class ChangeBillingComponent implements OnInit, OnDestroy {
      * @memberof ChangeBillingComponent
      */
     public ngOnInit(): void {
+        this.currentTimestamp = this.generalService.getTimesStamp();
```
Comment (ravinderwalkover) [inline apps/web-giddh/src/app/subscription/change-billing/change-billing.component.ts]:
```suggestion
        this.currentTimestamp = this.generalService.getTimeStamp();
```

**Review by ravinderwalkover — CHANGES_REQUESTED** [review]

File: apps/web-giddh/src/app/subscription/buy-plan/buy-plan.component.html (hunk 1 above)
Comment (dilpreetsinghofficial) [reply apps/web-giddh/src/app/subscription/buy-plan/buy-plan.component.html]: Autocomplete uses a string value, so I assigned a dynamic value.

File: apps/web-giddh/src/app/subscription/buy-plan/buy-plan.component.html [hunk 5]
```diff
@@ -709,10 +716,11 @@ <h4 class="font-16 strong d-flex align-items-center mb-3 plan-summary-heading">
                                                 <mat-form-field appearance="outline" class="w-100">
                                                     <mat-label>{{ commonLocaleData?.app_address }}</mat-label>
                                                     <textarea
+                                                        [name]="'subscriptionTextarea' + currentTimeStamp"
```
Comment (ravinderwalkover) [inline apps/web-giddh/src/app/subscription/buy-plan/buy-plan.component.html]:
```suggestion
                                                        [name]="'subscriptionAddress' + currentTimeStamp"
```

**Review by ravinderwalkover — CHANGES_REQUESTED** [review]

**Review by ravinderwalkover — APPROVED** [review]

**Pull request merged**